src/rgw/rgw_sal_motr.cc

```diff
@@ -721,6 +721,9 @@ int MotrBucket::put_info(const DoutPrefixProvider *dpp, bool exclusive, ceph::re
   bufferlist bl;
   struct MotrBucketInfo mbinfo;
   string tenant_bkt_name = get_bucket_name(info.bucket.tenant, info.bucket.name);
+  
+  // Set exclusive to false for put bucket tags
+  exclusive = (attrs.find(RGW_ATTR_TAGS) == attrs.end());
```

Comment:
@selvakumaar5496 can you please confirm if this agreed solution or temporary workaround?
If it's a workaround, please add a TODO.

Reply:
@jjxsg its discussed solution, plus tested with different APIs to confirm any possible issues, please check in [PR description](https://github.com/Seagate/cortx-rgw/pull/199#issue-1231077520).
exclusive=false needed to make update=true @ line 737, to update existing bucket entry for tags update. 

Same way different attributes seen detected, like seen determining for RGW_ATTR_ACL in rgw_op_get_bucket_policy_from_attr() in rgw_sal_rados.cc, and likewise for other attributes across the code.

Reply:
I have updated the PR as discussed with @sumedhak27 and @sachinpunadikar . We can consider this as a solution if sachin and sumedh agrees
cc: @jjxsg

Reply:
Thanks for clarification @cdeshmukh @selvakumaar5496